src/main/java/com/neonlab/product/utils/JsonUtil.java

```diff
@@ -0,0 +1,15 @@
+package com.neonlab.product.utils;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.neonlab.common.expectations.ServerException;
+
+
+public class JsonUtil {
```

Comment:
This is an functionality that will be used across microservices and hence please move this logic to `common`

*Note*: 
Any functionality that can be used across microservices should be in `common`.